### PR TITLE
Add support for etcd config file

### PR DESCRIPTION
**Pull request by glerchundi**
Mimics kube-apiserver behaviour and allows to access a TLS secured etcd too.

Signed-off-by: Gorka Lerchundi Osa <glertxundi@gmail.com>

### CODE REVIEW

File: pod-master/podmaster.go [hunk 1]
```diff
@@ -35,14 +35,15 @@ import (
...
-	etcdServers string
-	key         string
-	whoami      string
-	ttl         uint64
-	src         string
-	dest        string
-	sleep       time.Duration
-	lastLease   time.Time
+	etcdServers    string
+	etcdConfigFile string
```
Comment (wojtek-t) [inline pod-master/podmaster.go]: My feeling was that we are going to get rid of it in apiserver.

@timothysc @lavalamp - am I right?

**Comment by glerchundi** [comment]
I signed it!

File: pod-master/podmaster.go (hunk 1 above)
Comment (glerchundi) [inline pod-master/podmaster.go]: Ok, If you all agree I'll close this PR.

Whatever you guys decide it's ok but having a tls capable podmaster is absolutely necessary :-)

Comment (timothysc) [inline pod-master/podmaster.go]: Yes, it's removed in the new client patches.

**Pull request closed without merging**